integration_tests/src/main/python/orc_write_test.py

```diff
@@ -166,3 +167,30 @@ def create_empty_df(spark, path):
         lambda spark, path: spark.read.orc(path),
         data_path,
         conf={'spark.rapids.sql.format.orc.write.enabled': True})
+
+@allow_non_gpu('DataWritingCommandExec')
+@pytest.mark.parametrize("path", ["", "kms://http@localhost:9600/kms"])
+@pytest.mark.parametrize("provider", ["", "hadoop"])
+@pytest.mark.parametrize("encrypt", ["", "pii:a"])
+@pytest.mark.parametrize("mask", ["", "sha256:a"])
+@pytest.mark.skipif(is_databricks104_or_later(), reason="The test will fail on Databricks10.4 because `HadoopShimsPre2_3$NullKeyProvider` is loaded")
```

Comment:
does it make sense to try to gset the provider to something else to be able to test behavior on DB?

Reply:
I tried setting it to `test:///` which is what spark sets in their test code, but that doesn't work.

I am following the Spark teams lead on just skipping these tests although their tests are in Scala so they have control over skipping it only if the provider doesn't have test keys. I am just skipping them for DB10.4 only.

Reply:
Other combinations of the conf have resulted in similar errors

Reply:
So, I am not saying DB doesn't support encryption but when I set the provider to `unknown` I get errors saying that the provider is required. When I set it to `hadoop` I get the following error
```
Cause: org.apache.spark.SparkException: Job aborted due to stage failure: Task 0 in stage 1.0 failed 1 times, most recent failure: Lost task 0.0 in stage 1.0 (TID 1) (localhost executor driver): java.lang.IllegalArgumentException: Unknown key pii
```
If I set it to `aws` I get the error that the provider url is unreachable or something along those lines. I imagine an encryption server needs to be setup before encryption can be actually tested on DB.

Reply:
ok lets leave it as skip for 22.06, can you file a follow on issue to see if we can find a way to test on db 10.4.  We can discuss on the issue to see if we think its needed.

Reply:
Here is the follow-on. Please add to the issue or comment on it if something needs to be added 

#5722